filesmanager/filesmanager.py

```diff
@@ -130,12 +132,18 @@ class FilesManagerXBlock(XBlock):
         help="List of content paths to be displayed in the Files Manager."
     )
 
-    temporary_uploaded_files = Dict(
-        default={},
+    temporary_uploaded_files = List(
+        default=[],
         scope=Scope.settings,
         help="List of temporary uploaded files."
     )
 
+    source_keys = Dict(
+        default={},
+        scope=Scope.settings,
+        help="List of source keys."
```

Comment:
what are source keys?

Reply:
Is a list that allows us to track whether a course has been published. I think we can rename this variable to: `published_assets`